chain/network/src/peer_manager.rs

```diff
@@ -310,52 +131,36 @@ impl PeerManagerActor {
             },
         );
 
-        let wait_before_announce_seconds = 1;
-        ctx.run_later(Duration::from_secs(wait_before_announce_seconds), move |act, _ctx| {
-            // Sync routes to known account ids with new peer.
-            for mut announcement in
-                act.routing_table.account_peers.values_mut().map(|entry| entry.next_best_route())
-            {
-                // Update route with our information in case we are not the source.
-                if announcement.header().peer_id != act.peer_id {
-                    announcement.extend(act.peer_id.clone(), &act.config.secret_key);
-                }
-                actix::spawn(
-                    addr.send(SendMessage { message: PeerMessage::AnnounceAccount(announcement) })
-                        .map_err(|e| error!(target: "network", "{}", e))
-                        .map(|_| ()),
-                );
-            }
+        // TODO(MarX): Trigger sync actions after a peer is added successfully regarding networking
+        //  (Write specification about the actions)
 
-            // Send information from myself, in case it exists.
-            if let Some(my_announcement) = act.routing_table.me.clone() {
-                actix::spawn(
-                    addr.send(SendMessage {
-                        message: PeerMessage::AnnounceAccount(my_announcement),
-                    })
-                    .map_err(|e| error!(target: "network", "{}", e))
-                    .map(|_| ()),
-                );
-            }
-        });
+        self.routing_table.add_connection(source, target);
     }
 
+    /// Remove a peer from the active peer set. If the peer doesn't belong to the active peer set
+    /// data from ongoing connection established is removed.
     fn unregister_peer(&mut self, peer_id: PeerId) {
         // If this is an unconsolidated peer because failed / connected inbound, just delete it.
         if self.outgoing_peers.contains(&peer_id) {
             self.outgoing_peers.remove(&peer_id);
             return;
         }
         self.active_peers.remove(&peer_id);
-        self.routing_table.remove(&peer_id);
+
+        // TODO(MarX): Trigger actions after a peer is removed successfully regarding networking
```

Comment:
Create a github issue and reference it here and in other places where you refer to it

Reply:
done with every TODO in networking.